### DISCUSSION TITLE
PWA Integration with linux app launchers/start menu (.desktop files)

### DISCUSSION THREAD

**Discussion by Cyanide-1** [Floorp-Projects/Floorp#903] [Feature Request]
I love the PWA Feature, but there is missing functionality: app launcher integration

Normally on most operating systems, PWA's integrate with the system's app launcher, on linux this is done through .desktop files (the cross-desktop linux standard), on floorp the PWA's don't do this, i suggest adding .desktop file integration so that all web apps can integrate with linux app launchers, thank you for reading!

**Comment by cameronj86** [comment]
Let me know if this occurs on native installs. If so, I can close out my ticket ( #853 ) and redirect attention to this one

**Comment by CutterKnife** [comment]
Is it related to this issue?
https://github.com/Floorp-Projects/Floorp/issues/909
